prime-router/src/test/kotlin/ElementTests.kt

```diff
@@ -178,6 +178,15 @@ internal class ElementTests {
         one.toNormalized("     ").run {
             assertThat(this).isEqualTo("")
         }
+
+        // Hours with single digits
+        val testTimes = listOf(
+            "3/30/1998 9:35", "3/30/1998 09:35", "1998/3/30 9:35", "1998/3/30 09:35", "19980330 9:35:00",
+            "19980330 09:35:00", "1998-03-30 9:35:00", "1998-03-30 09:35:00"
```

Comment:
All of these test dates are before noon. Please add some that are afternoon to make sure they work as well.

Reply:
Done